src/main/java/org/apache/sling/feature/cpconverter/artifacts/LocalMavenRepositoryArtifactsDeployer.java

```diff
@@ -99,4 +92,17 @@ public void deploy(@NotNull ArtifactWriter artifactWriter, @NotNull ArtifactId i
         }
     }
 
+    public static String getNameFromArtifactId(ArtifactId id) {
+        StringBuilder nameBuilder = new StringBuilder()
```

Comment:
Why not use ArtifactId.toMvnName() ?

Reply:
I just extracted the existing code into a new method, but even better to reuse the one from `ArtifactId`.

Reply:
Done in https://github.com/apache/sling-org-apache-sling-feature-cpconverter/pull/70/commits/e569b904545f7b58e81e1242ed2bd87eb58a4f55.